tools/Makefile

```diff
@@ -25,7 +25,7 @@ endif
 
 all: kaldi.done python check_install
 
-python: venv warp-ctc.done warp-transducer.done chainer_ctc.done
+python: venv warp-ctc.done warp-transducer.done chainer_ctc.done kenlm.done
```

Comment:
https://github.com/espnet/espnet/pull/1946/checks?check_run_id=694960347 
As seen in this CI error, I concern that installing kenlm seems so tough. This is because it requires Boost. I think it should be optional. Consider `pip install https://github.com/kpu/kenlm/archive/master.zip` for python installation. https://github.com/kpu/kenlm

Reply:
I think the best way to install kenlm python binding is adding the github URL to `setup.py` like https://github.com/espnet/espnet/blob/66f0a1263e05848482377ad6c515b03ccb20d594/setup.py#L48-L49

Reply:
but it needs "lmplz" and "build_binary"

Reply:
@ShigekiKarita Now we can't use direct installation from url in setup.py, `@git+https://`, because pypi server doesn't allow it. ( I registered espnet to pypi recently )

@qmpzzpmq 
- Change the installation for kenlm to an option.
- Remove the dependencies on "lmplz" and "build_binary" on the test. It can be avoided by using stored arpa and bin file.
- Install kenlm by `pip install https://github.com/kpu/kenlm/archive/master.zip` for the unittest: Here https://github.com/espnet/espnet/blob/develop/ci/install.sh